src/citrine/resources/data_concepts.py

```diff
@@ -482,7 +513,7 @@ def async_update(self, model: ResourceType, *,
         job_id = response_json["job_id"]
 
         if wait_for_response:
-            self.poll_async_update_job(job_id, timeout=timeout,
+            self.poll_async_update_job(job_id=job_id, timeout=timeout,
                                        polling_delay=polling_delay)
```

Comment:
Are we intending to obsolete polling with just a job_id?  What's the point of this change?

Reply:
What do you mean? It's just switching to use a keyword argument invocation, as we advocate almost everywhere.